main.cpp

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+#include <ctype.h>
+#include "quadratic_equation.h"
+
+int main() {
+    double first_par = 0;
+    double second_par = 0;
+    double third_par = 0;
+
+    printf("Enter the first second and third parameters of the quadratic equation separated by a space:\n");
+
+    int result_scanf = 0;
+
+    do {
+        result_scanf = scanf("%lf%lf%lf", &first_par, &second_par, &third_par);
+
+        if (result_scanf == 0) {

```

Comment:
If you use while loop instead of do-while, then there is no need for this condition.